schedule/schedule-appengine/src/main/client/jasify/admin/activity/admin-subscribers.js

```diff
@@ -30,7 +33,8 @@
         }
 
         function fail(r) {
-            vm.alert('danger', 'Failed: ' + r.statusText);
+            var translation = $translate('FAILED');
+            vm.alert('danger', translation + ': ' + r.statusText);
         }
```

Comment:
Are we going to translate the "statusText" on the backend?  One option would be to say `$trasnlate(r.statusText)` but then any other client of the API would need to implement translations.